timemachine/fe/plots.py

```diff
@@ -315,7 +315,8 @@ def plot_hrex_transition_matrix(
     transition_rate: NDArray,
```

Comment:
(not introduced in this PR, but I just noticed a small naming imprecision here: these are transition _probabilities_, not transition _rates_, correct? (if these were rates, the diagonal would be negative))

Reply:
Good catch. Renamed in https://github.com/proteneer/timemachine/pull/1224/commits/60418a4b572efeb08c5d7d46ec4e934dee5fc77e